docs/source/composer_model.rst

```diff
@@ -210,42 +210,19 @@ and make it compatible with our trainer.
     from torchmetrics import Accuracy
     from torchmetrics.collections import MetricCollection
 
-    from composer import ComposerModel
-    from composer.models.nlp_metrics import LanguageCrossEntropyLoss
+    from composer.models.huggingface import HuggingFaceModel
+    from composer.metrics.nlp import LanguageCrossEntropy
 
-    class ComposerBERT(ComposerModel):
-        def __init__(self, num_labels):
-            super().__init__()
-            # huggingface model
-            self.model = AutoModelForSequenceClassification.from_pretrained(
+    # huggingface model
+    model = AutoModelForSequenceClassification.from_pretrained(
                             'bert-base-uncased',
-                            num_labels=num_labels
-                        )
-
-            # Metrics
-            self.train_loss = LanguageCrossEntropyLoss()
-            self.val_loss = LanguageCrossEntropyLoss()
-            self.train_acc = Accuracy()
-            self.val_acc = Accuracy()
-
-        def forward(self, batch):
-            outputs = self.model(**batch)
-            return outputs
-
-        def loss(self, outputs, batch):
-            return outputs['loss']  # huggingface models output a dictionary
-
-        def validate(self, batch):
-            labels = batch.pop('labels')
-            output = self.forward(batch)
-            output = output['logits']
-            return output, labels
+                             num_labels=num_labels
 
-        def metrics(self, train: bool = False):
-            if train:
-                return MetricCollection([self.train_loss, self.train_acc])
-            return MetricCollection([self.val_loss, self.val_acc])
+    # list of torchmetrics
+    metrics = [LanguageCrossEntropy(vocab_size=tokenizer.vocab_size), Accuracy()]
```

Comment:
`tokenizer` isn't defined (which could be okay, but we should be explicit about this so user don't spend time searching for it in the rest of the code like I just did). I would vote for a working code though.

Reply:
agreed, will change all examples to be working, minimal examples.